packages/deploy-tooling/test/unit/base/validate.test.ts

```diff
@@ -76,6 +76,36 @@ describe('deploy-test validation', () => {
             expect(summaryStr).toContain(`${t('deploy.abapInvalidAppName', { prefix })}`);
         });
 
+        test('Skip validating url if destination is provided', async () => {
+            const name = 'nslooooooooooooooooooooog/ZAPP1';
+            const prefix = 'Z';
+            const destination = 'TestDestination';
+            mockedAdtService.listPackages.mockResolvedValueOnce(['TESTPACKAGE', 'MYPACKAGE']);
+            mockedAdtService.getTransportRequests.mockResolvedValueOnce([
+                { transportNumber: 'T000001' },
+                { transportNumber: 'T000002' },
+                { transportNumber: 'T000003' }
+            ]);
+            mockedAdtService.getAtoInfo.mockResolvedValueOnce({
+                developmentPrefix: prefix
+            });
+
+            const output = await validateBeforeDeploy(
+                {
+                    app: { ...app, name },
+                    target: { ...target, destination, url: undefined }
+                },
+                mockedProvider as any,
+                nullLogger
+            );
+            expect(output.result).toBe(false);
+            const summaryStr = formatSummary(output.summary);
+            expect(summaryStr).toContain(`${red('×')} ${t('deploy.invalidAppNameMultipleReason')}`);
```

Comment:
We're asserting for the error message already. Do we need to assert it's `red('x')` as well?